File: vue/src/api/es-task.js
import request from '@/utils/request'

const api = '/api/es_task/'

export function ListAction(data) {
  return request({
    url: api + 'ListAction',
    method: 'post',
    data
  })
}
export function CancelAction(data) {
  return request({
    url: api + 'CancelAction',
    method: 'post',
    data
  })
}

File: pkg/web/esTask.go
package web

// ES 任务 路由
func (this *WebServer) runEsTask() {

	const AbsolutePath = "/api/es_task"
	group := this.engine.Group("ES映射", AbsolutePath)
	{
		group.POST(false, "查看任务列表", "/ListAction", this.esTaskController.ListAction)
		group.POST(false, "取消任务", "/CancelAction", this.esTaskController.CancelAction)
	}
}
